TYPO3.Neos/Classes/TYPO3/Neos/Service/View/NodeView.php

```diff
@@ -237,7 +237,9 @@ public function collectParentNodeData(NodeInterface $rootNode, array $nodes)
                     $collectTreeNodeData($children, $childNode);
                 }
             }
-            $treeNodes[] = $self->collectTreeNodeData($node['node'], true, $children, $children !== array(), isset($node['matched']));
+            if (isset($node['node']) && $node['node'] instanceof NodeInterface) {
```

Comment:
I see, but you can actually just disable the other jobs in the repair job and only check for orphan nodes since it doesn't create actual nodes.

Anyway I agree it should work regardless, but we should figure out why instead of just hot fixing it further down the pipe..